packages/transformers/svg-react/src/SvgReactTransformer.js

```diff
@@ -17,7 +17,13 @@ function getComponentName(filePath) {
 }
 
 export default (new Transformer({
-  async transform({asset}) {
+  async loadConfig({config}) {
+    let {contents} =
+      (await config.getConfig(['.svgrrc.json', '.svgrrc'])) || {};
+
+    return contents;
```

Comment:
Nit: this might be easier to read:

```suggestion
    let conf = await config.getConfig(['.svgrrc.json', '.svgrrc'])
    return conf?.contents;
```